fastlane/metadata/android/en-US/changelogs/983.txt

```diff
@@ -0,0 +1,9 @@
+New fast forward / rewind behaviour
```

Comment:
```suggestion
New fast forward / rewind behaviour and UI
```

Reply:
@opusforlife2 somehow it looks better the other way around:
```suggestion
New fast forward / rewind UI and behavior
```